bundle/Resources/views/Content/content_form.html.twig

```diff
@@ -0,0 +1,23 @@
+{% macro display_form(form) %}
```

Comment:
Could we put this file in a `macros` folder in order to reflect it only contains a macro and not an actual displayable form ?